src/NuGet.Clients/NuGet.Console/NuGet.Console.csproj

```diff
@@ -147,6 +147,7 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" NoWarn="NU1604" />
```

Comment:
Declare it in a similar way as source link: https://github.com/NuGet/NuGet.Client/blob/dev/build/common.project.props#L278-L280